inc/modules/class-image.php

```diff
@@ -50,4 +50,19 @@ public function get_json() {
 		}, $data['image'] );
 		return $data;
 	}
+
+	public function get_rest_links() {
+		$data = parent::get_json();
+
+		if ( ! $data['image'] ) {
```

Comment:
Should we have a filter here for any additional / custom modules that may contain attachments of any type or links to other content? I'd argue this is specifically for the images right now so `get_rest_links()` is a bit too generic as a name.